registry.json

```diff
@@ -4017,18 +4017,31 @@
    "github": "sephirot1024"
   },
   {
-   "ticker": "CBFR",                                                      
-   "poolId": "d6095a71081323dbc614d974f6159c37cd5f4133c8830172f877ca51",  
-   "social": {                                                            
+   "ticker": "CBFR",
+   "poolId": "d6095a71081323dbc614d974f6159c37cd5f4133c8830172f877ca51",
+   "social": {
      "website": "",
      "twitter": "CardanoFR",
       "facebook": "",
       "instagram": "",
       "youtube": ""
    },
-   "telegram": "missawCBFR",                                             
-   "operatorName": "Wassim",                                   
-   "github": "missawZero"                                                     
+   "telegram": "missawCBFR",
+   "operatorName": "Wassim",
+   "github": "missawZero"
+  },
+  {
+   "ticker": "GLOW",
+   "poolId": "5dc1f763f14baaabc8d14aa9c0717e8d4662e4d16b2e10095d7ec78b",
+   "social": {
+     "website": "https://cardano.xbglowx.com",
+     "twitter": "CardanoGlow",
```

Comment:
There is no activity on this Twitter account and the website doesnt seem to load anything for me at present. We ask that all members have some form of social media presence. Please can you add some activity and I will re-review? Thanks

Reply:
Thanks for the review.

Still working on getting everything configured, hence the lack of Twitter activity and https://cardano.xbglowx.com not loading. 

I will ping once I have fixed your concerns.

Reply:
Can you please have another look?

Reply:
Sorry to ping again. Ready for my close up.